code/index.html

```diff
@@ -4,27 +4,274 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <link rel="stylesheet" href="main.css" />
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
+    <!-- Header with full width video & nav links to login -->
+    <header class="hero">
+      <video class="hero__video" playsinline autoplay muted loop>
+        <source src="./assets/videos/adventures.mp4" type="video/mp4" />
+        <source src="./assets/videos/adventures.webm" type="video/webm" />
+        Sadly your browser not support HTML 5 video.
+      </video>
+      <h1 class="hero__title">
+        Let the <br />
+        <span>ADVENTURE</span> <br />
+        Begin
+      </h1>
+      <nav class="hero__nav">
+        <button class="button hero__button">LOGIN</button>
+      </nav>
+    </header>
 
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+    <!-- Section containing the form -->
+    <section class="forms">
+      <div class="content__container">
+        <h2 class="content__title">Book your adventure!<span>Learn the wilderness</span></h2>
+        <!-- The Form -->
+        <form class="content" action="https://httpbin.org/anything" method="post">
+          <!-- Form part 1 -->
+          <!-- Contact information -->
+          <fieldset class="form__section">
+            <legend class="form__section-title">Who is booking?</legend>
+            <!-- Name (text) -->
+            <label class="form__input-wrapper" for="name">
```

Comment:
If we decide to nest input inside label, then we don't need to add for - id pair. More details in Week 3 Master document